pkgs/development/compilers/ponyc/default.nix

```diff
@@ -24,9 +24,9 @@ stdenv.mkDerivation (rec {
   };
 
   ponygbenchmark = fetchurl {
-    url = "https://github.com/google/benchmark/archive/v1.5.0.tar.gz";
-    sha256 = "06i2cr4rj126m1zfz0x1rbxv1mw1l7a11mzal5kqk56cdrdicsiw";
-    name = "v1.5.0.tar.gz";
+    url = "https://github.com/google/benchmark/archive/v1.5.2.tar.gz";
+    sha256 = "033hgsc76bcii1gwd8hdc7z1iqdnggk4f8cq0hzh98dsjsvxmjyw";
+    name = "v1.5.2.tar.gz";
```

Comment:
```suggestion
```

Reply:
Ah, because name: isn't used I'm guessing?

If that's the case I can also remove the one on L108.

I'll update both of those and push - thanks.

Reply:
I've also removed propagatedBuildInputs.  It is no longer required.